src/library/scala/collection/immutable/NumericRange.scala

```diff
@@ -15,37 +15,35 @@ package scala.collection.immutable
 import scala.collection.Stepper.EfficientSplit
 import scala.collection.{AbstractIterator, AnyStepper, IterableFactoryDefaults, Iterator, Stepper, StepperShape}
 
-/** `NumericRange` is a more generic version of the
-  *  `Range` class which works with arbitrary types.
-  *  It must be supplied with an `Integral` implementation of the
-  *  range type.
+/** `NumericRange` is a more generic version of the `Range` class which works with arbitrary types. It must be supplied
+  * with an `Integral` implementation of the range type.
```

Comment:
The original ("correct") indentation is seen at https://github.com/scala/scala/blob/2.12.x/src/library/scala/collection/immutable/NumericRange.scala#L20

Unfortunately, when they "imported" the strawman files, all the comments were corrupted to the weird style.

The unweird style is uniformly preferred on Scala 3 (https://github.com/lampepfl/dotty/blob/main/library/src/scala/CanEqual.scala#L10) so I would normalize to that form when incurring whitespace edits.

Reply:
Sorry folks, my bad! I wasn't planning to change the whitespace and layout of the entire class. I must've accidentally hit an autoformatter shortcut and not noticed during the commit. I'll roll back all the style changes outside of my new functions.